### DISCUSSION TITLE
Is possible to retrieve inferred type?

### DISCUSSION THREAD

**Discussion by barbulescu** [LemonAppDev/konsist#1627] [Ask a Question]
### Wanted end result.

`val foo = buildFoo()`

Is possible to retrieve inferred type in this case?

Property type comes as null in this case.

### What you tried so far.

- checked documentation
- debugged and try to find any field having this information

**Comment by igorwojda** [comment]
ATM Konsist does not allow to verity inferred types (only actual text present in Kotlin files).
Perhaps in the future we will be able to utilize Kotlin Compiler plugin to get more information about type system.
